packages/web-react/src/components/Tag/Tag.tsx

```diff
@@ -2,35 +2,49 @@ import React, { ElementType, JSXElementConstructor } from 'react';
 import classNames from 'classnames';
 import { filterProps } from '../../utils/filterProps';
 import { compose } from '../../utils/compose';
-import { applyColor, applyTheme } from '../../utils/classname';
+import { applyColor, applySize, applyTheme } from '../../utils/classname';
 import { useClassNamePrefix } from '../../hooks/useClassNamePrefix';
 import { ChildrenProps } from '../../types';
 
 type Color = 'default' | 'informative' | 'success' | 'warning' | 'danger';
 
+type Size = 'xsmall' | 'small' | 'medium';
```

Comment:
This is another problem with implementation vs design. Cannot be consistent in `size` prop all across the component. Please use `TagSize` instead to be specific. This should also apply to `Color` but it is not exported. Also `Size` is not also exported so maybe it do not matter.